hypermode-chat/src/app/apollo-wrapper.tsx

```diff
@@ -0,0 +1,35 @@
+"use client"
+
+import { ApolloClient, ApolloProvider, InMemoryCache, HttpLink, from } from "@apollo/client"
+import { setContext } from "@apollo/client/link/context"
+import { RetryLink } from "@apollo/client/link/retry"
+
+export function ApolloWrapper({ children }: React.PropsWithChildren<{}>) {
+  const httpLink = new HttpLink({
+    uri: process.env.NEXT_PUBLIC_GRAPHQL_API_URL || "http://localhost:8686/graphql",
```

Comment:
@rderbier, you can connect to the chatkit backend to test this out, take the URL and key from here:

https://hypermode.com/hypermode/apps/chatkit